pkg/util/resource/scanner.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 const (
-	heapMetricName = "/memory/classes/Heap/objects:bytes"
+	heapMetricName = "/memory/classes/heap/objects:bytes"
```

Comment:
Would `Test_HeapScanner` test fail using the old heap metric name?

Reply:
yes, it will fail with a wrong metric name